extensions/markdown-language-features/src/preview/preview.ts

```diff
@@ -679,6 +686,12 @@ export class DynamicMarkdownPreview extends Disposable implements IManagedMarkdo
 		});
 	}
 
+	openImage(imagePath: string): void {
```

Comment:
Let's move this code into the command. The second argument to `vscode.open` is a `ViewColumn` if you need to open in a specific editor group